src/io.rs.bk

```diff
@@ -0,0 +1,80 @@
+//! Operations that interact with the file system.
+
```

Comment:
Delete this file

Reply:
will do, I'm guessing that came from using rustfmt.